contracts/Event.sol

```diff
@@ -189,12 +180,24 @@ contract CyberCore is CyberCoin {
 
     _freeze(_tokenId);
 
-    if (events[_tokenId].ticketPrice > 0) {
+    if (events[_tokenId].ticketPrice > 0 wei) {
       ownerOf(_tokenId).transfer(
         events[eventId(_tokenId)].ticketPrice.div(100).
-        mul(100 - events[eventId(_tokenId)].
-        speakersPercent - events[eventId(_tokenId)].
-        ownerPercent)
+        mul(events[eventId(_tokenId)].cashbackPercent)
+      );
+      events[eventId(_tokenId)].paidAmount = (
+        events[eventId(_tokenId)].paidAmount.
+        sub(
+          events[eventId(_tokenId)].ticketPrice.div(100).
+          mul(events[eventId(_tokenId)].cashbackPercent)
+        )
+      );
+      paidAmountOf[ownerOf(_tokenId)][eventId(_tokenId)] = (
+        paidAmountOf[ownerOf(_tokenId)][eventId(_tokenId)].
+        sub(
+          events[eventId(_tokenId)].ticketPrice.div(100).
+          mul(events[eventId(_tokenId)].cashbackPercent)
+        )
```

Comment:
Add a lot of business logic that is not connected to declared issue and pl topic